package.json

```diff
@@ -15,7 +14,7 @@
     "prebuild:prod": "npm run clean",
     "build:dev": "run-p plugins:dev copy:php sass watch js:watch",
     "build:prod": "run-s plugins:build copy js:prod 'sass -- --style compressed'",
-    "serve:dev": "docker-compose up",
+    "serve:dev": "docker compose up --abort-on-container-exit || docker compose down",
```

Comment:
It seems like `post*` scripts work roughly the same as chaining commands with `&&` in the terminal, so when a non-zero exit code (or a Ctrl+C) happens, it would stop there. By using the `||` operator, the "failed" command ends and the "fallback" command runs.